display-scene/src/main/res/values/colors_blue_green.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="colorPrimary">#2196F3</color>
+  <color name="colorPrimaryDark">#1976D2</color>
+  <color name="colorPrimaryLight">#BBDEFB</color>
+  <color name="colorAccent">#4CAF50</color>
+  <color name="colorPrimaryText">#212121</color>
+  <color name="colorSecondarytext">#757575</color>
+  <color name="colorIcons">#FFFFFF</color>
+  <color name="colorDivider">#BDBDBD</color>
```

Comment:
Are the values other than colorPrimary, colorPrimaryDark and colorAccent actually used?